src/App.css

```diff
@@ -1,3 +1,210 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Mochiy+Pop+One&family=Roboto+Slab:wght@300;400;500;600;900&family=Source+Code+Pro:wght@400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Mochiy+Pop+One&family=Montserrat:ital,wght@0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700&family=Roboto+Slab:wght@300;400;500;600;900&family=Source+Code+Pro:wght@400;500;600;700&display=swap');
+
 body {
+  background-color: #fafafa;
+  font-family: 'Roboto Slab', serif;
+}
+
+nav {
+  display: flex;
```

Comment:
- [ ] As a best practice, keeping your `css` dry, rather than using the same `css rule` repetitively,  for multiple elements, it will be nice to use a `class attribute` instead. Checkout this [article ](https://speckyboy.com/good-bad-css-practices/)and kindly fix that.

Reply:
updated code to use DRY principles